sdk/typescript/src/zklogin/bcs.ts

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { BCS } from '@mysten/bcs';
+
+import { bcsRegistry } from '../bcs/index.js';
+
+export const zkLoginBcs = new BCS(bcsRegistry);
+
+zkLoginBcs.registerStructType('ZkLoginSignature', {
```

Comment:
can we use the new BCS api for this instead? https://sui-typescript-docs.vercel.app/bcs